cypress/integration/ShowCertification.js

```diff
@@ -57,15 +57,15 @@ describe('A certification,', function() {
         .and(
           'match',
           // eslint-disable-next-line max-len
-          /https:\/\/www\.linkedin\.com\/profile\/add\?startTask=CERTIFICATION_NAME&name=Legacy Front End&organizationId=4831032&issueYear=\d\d\d\d&issueMonth=\d\d?&certUrl=https:\/\/freecodecamp\.org\/certification\/developmentuser\/legacy-front-end/
+          /'https:\/\/www\.linkedin\.com\/profile\/add\?startTask=CERTIFICATION_NAME&name=Legacy Front End&organizationId=4831032&issueYear=\d\d\d\d&issueMonth=\d\d?&certUrl=http:\/\/localhost:8000\/certification\/developmentuser\/legacy-front-end/
```

Comment:
Just to mention: I am going around changing values like `localhost:8000` to `${Cypress.env('BASE_URL')}`. This way, it defaults to whatever is set in either `cypress.json` or what is set in `.gitpod.yml`